### ISSUE TITLE
SRW - cannot self-extract simulation to beta server

### ISSUE THREAD

**Issue by mrakitin** [radiasoft/sirepo#852]
I was trying to self-extract a simulation exported from our local server to beta.sirepo.com, but it failed with the error at the page https://beta.sirepo.com/import-archive:
```
Not Found

The requested URL was not found on the server. If you entered the URL manually please check your spelling and try again.
```

The situation is similar on alpha server - it fails from the first attempt, but from the second the extraction was successful (same on beta). The file is attached - [Sirepo_paper_html.zip](https://github.com/radiasoft/sirepo/files/1245696/Sirepo_paper_html.zip).

P.S. After the second attempt there is a notification:
```
Font download has been disabled for this browser. Application icons will not be displayed correctly as a result. Either enable font download on the Internet Options / Security Settings menu or switch to a different browser, such as Google Chrome or Microsoft Edge.
```
Is it something expected? I use Firefox 54.0.1 (64-bit) + Adblock.

**Comment by mrakitin** [comment]
OK, I investigated a bit, and it appears to fail when I provide `http` address instead of `https`. When using https://beta.sirepo.com everything runs fine.

@robnagler, it there anything that could be fixed on the server side? Thanks.

**Comment by moellep** [comment]
The app could ping the target server to see if it is listening on HTTPS and automatically change the request. Alternately, it could check if the target is *.sirepo.com and automatically use HTTPS.

**Comment by mrakitin** [comment]
Good idea.

**Comment by moellep** [comment]
fixed with 2d4aa75
